### ISSUE TITLE
Include command line tool

### ISSUE THREAD

**Issue by Sturwandan** [nayuki/QR-Code-generator#132]
I would like to generate QR code from bash, but it seems that there is no qr encoder executable. There is only a demo in /c/, however I wanted something like dtmxwrite from https://github.com/dmtx/dmtx-utils which allows to generate code with my text instead of predefined text and save it to image file if necessary. 

It also would be nice if it had option to select characters for blank and filled areas, instead of hardcoded ## and double space, for I might want to use 1 and 0 to dump it into netpbm file, or ██ to scan directly from terminal.

The online demo on your website looks nice, but I'd rather not depend from a web site.

**Comment by nayuki** [comment]
Sorry, please do it yourself. At the moment I am not interested in writing and maintaining a fully featured command-line programs in every language.

> generate code with my text instead of predefined text ... instead of hardcoded ## and double space

Modifying the demo program to suit your needs isn't hard. Use `argv` as the input text. Delete the vast majority of the demo functions.

> save it to image file

I don't want to deal with the selection and maintenance of third-party libraries.

> option to select characters for blank and filled areas

Options upon options.

> I'd rather not depend from a web site

Compile TypeScript to JavaScript and use the local web page: https://github.com/nayuki/QR-Code-generator/tree/master/typescript-javascript

**Comment by Sturwandan** [comment]
Thank you for your answer.

I wrote this bug report before discovering that my OS already had ``qrencode`` executable.
Though, it turned out, that I didn't really need anything at all, because I wanted to also have an aztec encoder and only one I found for Linux was zint, which also supports all other types of barcodes, including QR and DataMatrix, so I don't really need anything else anymore for now.

> At the moment I am not interested in writing and maintaining a fully featured command-line programs in every language.
I think one in C language would be sufficient.

> I don't want to deal with the selection and maintenance of third-party libraries.

I guess that support for netpbm format would be sufficient. It's same as ASCII you output to terminal, except it has 0 and 1 instead of double whitespaces and double ## plus simple header with dimensions.

You don't have to add a command line tool if you don't want to, but I honestly kinda expected to find at least a simple one, when I downloaded and tried to compile the C language directory. But indeed, if there were no other generators, I'd have to modify the demo program as you suggested above.
